package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "strophejs-plugin-stream-management",
+  "version": "1.0.0",
+  "description": "Strophe.stream-management.js is a plugin that implements stream management on XMPP. ( XEP-0198 )",
+  "main": "strophe.stream-management.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/horodnicdragos/strophejs-plugin-stream-management.git"
```

Comment:
Please update all those links to the main repo.

Reply:
Hi, @sualko and sorry for the late reply. I've updated the url.